<launch>
    <node name="setParams" pkg="motor_control" type="params.py"/>
    <node name="dataFrameServer" pkg="motor_control" type="dataFrameServer.py"/>
    <node name="stepperImp" pkg="motor_control" type="stepperImp.py"/>
</launch>